app/javascript/components/search/controls/download/DownloadSelectionTable.js

```diff
@@ -175,7 +175,7 @@ const COLUMNS = {
   sequence: {
     title: 'Sequence',
     types: ['sequence_file'],
-    info: 'Sequence files, such as BAM or BAI files',
+    info: 'Sequence files, such as Fastq, BAM or BAI files',
```

Comment:
Blocker: per [convention](https://en.wikipedia.org/wiki/FASTQ_format), "Fastq" should be all-caps:

```suggestion
    info: 'Sequence files, such as FASTQ, BAM or BAI files',
```